### ISSUE TITLE
Obsidian Confluence Sync: wikilinks to non-image files cause the sync to fail

### ISSUE THREAD

**Issue by RaeASunshine** [markdown-confluence/markdown-confluence#364]
I have a number of documents i'm trying to sync with Confluence that essentially contain a couple of non-image files (mostly performance profiles) and then a text analysis of the document.  The intended workflow is to access the profile and then read through the supporting analysis.

When I try to sync these pages, I get the error:
![image](https://github.com/markdown-confluence/markdown-confluence/assets/455800/4cb8745a-ee8c-43b3-a1ee-d8f8fab1c667)

This stops the sync in it's tracks, and I have to remove these pages from the sync folder.

I want to push these pages to confluence and have the these non-image files just be uploaded as normal and linked into the page.

**Comment by andymac4182** [comment]
So you want them to upload as a normal attachment? This wasn't something I had tried previously but I imagine it won't be too hard to support.

**Comment by RoyAwesome** [comment]
Yeah.  A normal attachment if it's not an image, so that way you can just link to it.

**Comment by andymac4182** [comment]
Awesome :) Will have a look at what I need to tweak to get this working.
